.circleci/config.yml

```diff
@@ -833,6 +833,17 @@ jobs:
       <<: *infrastructure_config_environment
       PKG_TARGET_OS: ubuntu
     steps: *upload_latest_artifacts
+
+
+  verify_release_artifacts:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    steps:
+      - run:
+          name: Verify release artifacts
+          command: |
+            /infrastructure/scripts/check_release_artifacts.sh ${CIRCLE_TAG}
```

Comment:
could you please change this to relative path ?

Reply:
done in 1f61b41